test/test-integration/src/test/java/org/wso2am/micro/gw/tests/util/ApiProjectGenerator.java

```diff
@@ -85,13 +85,26 @@ public static String createApictlProjZip(String apiYamlPath, String swaggerPath,
             apiYamlFile);
         
         //swagger.yaml
-        Utils.copyFile(testResourcesPath + swaggerPath, apisZipPath + File.separator + definitions + File.separator +
-            openAPIFile);
+        if(swaggerPath != null){
+            Utils.copyFile(testResourcesPath + swaggerPath, apisZipPath + File.separator + definitions + File.separator +
+                    openAPIFile);
+        }
 
         if (certificatePath != null) {
             Utils.copyFile(testResourcesPath + certificatePath, 
                 apisZipPath + File.separator + endpointCertificates + File.separator + "backend.crt");
         }
+//        if(apiYamlPath != null){
```

Comment:
If the commented code needed let's remove it. If we want to keep it, let's state why we need to keep it.

Reply:
Fixed